src/ShortCode/Random.php

```diff
@@ -19,40 +19,35 @@
  *
  * @author Anis Uddin Ahmad <anis.programmer@gmail.com>
  */
-class Random extends Code
-{
-
-    /**
-     * Get a random code of fixed length.
-     *
-     * @param int $length  length of code, default 8
-     * @param string $outputFormat One of Code::FORMAT_* constants. Default Code::FORMAT_ALNUM
-     *
-     * @return string
-     */
-    public static function get($length = 8, $outputFormat = Code::FORMAT_ALNUM)
-    {
-        static::throwUnlessAcceptable($outputFormat, $length);
-
-        $number = rand(100, 900) . str_replace('.', '', microtime(true));
-        $output = self::convertBase($number, self::FORMAT_NUMBER, $outputFormat);
-
-        if(strlen($output) < $length) {
-            $output .= substr(str_shuffle($outputFormat.$outputFormat), 0, ($length - strlen($output)));
-        }
-        if(strlen($output) > $length) {
-            $output = substr($output, 0, $length);
-        }
-
-        return $output;
-    }
-
-    private static function throwUnlessAcceptable($type, $length)
-    {
-        if($length > 20) {
-            $typeName = self::getTypeName($type);
-            throw new UnexpectedCodeLength("Code length $length is not acceptable for $typeName");
-        }
-    }
+class Random extends Code {
+	/**
+	 * Get a random code of fixed length.
+	 *
+	 * @param int $length length of code, default 8
+	 * @param string $outputFormat One of Code::FORMAT_* constants. Default Code::FORMAT_ALNUM
+	 *
+	 * @return string
+	 */
+	public static function get( $length = 8, $outputFormat = Code::FORMAT_ALNUM ) {
+		static::throwUnlessAcceptable( $outputFormat, $length );
+
+		$number = rand( 100, 900 ) . str_replace( '.', '', microtime( true ) );
+		$output = self::convertBase( $number, self::FORMAT_NUMBER, $outputFormat );
+
+		if ( strlen( $output ) < $length ) {
+			$output .= substr( str_shuffle( $outputFormat . $outputFormat ), 0, ( $length - strlen( $output ) ) );
+		} else {
+			$output = substr( $output, 0, $length );
+		}
+
+		return $output;
+	}
+
+	private static function throwUnlessAcceptable( $type, $length ) {
+		if ( $length > 20 ) {
+			$typeName = self::getTypeName( $type );
+			throw new UnexpectedCodeLength( "Code length $length is not acceptable for $typeName" );
+		}
+	}
```

Comment:
According to https://www.php-fig.org/psr/psr-2/ , 
- The opening brace for the class and methods MUST go on its own line.
- Opening parentheses for control structures MUST NOT have a space after them, and closing parentheses for control structures MUST NOT have a space before. So,
```
if ( $length > 20 ) {
```
Should be 
```
if ($length > 20) {
```
- Same goes to function arguments, in both, declarations and calls
```
private static function throwUnlessAcceptable( $type, $length )
```

Should be 
```
private static function throwUnlessAcceptable($type, $length)
```